python/tests-integration/test_verifier.py

```diff
@@ -0,0 +1,46 @@
+import pathlib
+import subprocess
+import sys
+
+import pytest
+
+
+PLAYBOOK_DIRECTORY = pathlib.Path(__file__).parents[2].absolute() / "data" / "playbooks"
+
+
+@pytest.mark.parametrize(
+    "filename",
+    [
+        "insights_remove",
+        "document-from-hell",
+        "unicode",
+    ],
+)
+def test_official_playbook(filename: str):
+    """Test playbook signed by Red Hat's GPG key.
+
+    In this test, the official playbooks are verified against the GPG key
+    the application ships.
+    """
+    if filename == "unicode" and sys.version_info >= (3, 12):
+        raise pytest.xfail("CCT-644")
+
+    playbook_content: str = (PLAYBOOK_DIRECTORY / f"{filename}.yml").read_text()
+
+    result = subprocess.run(
+        [
+            "insights-ansible-playbook-verifier",
+            "--playbook",
+            f"{PLAYBOOK_DIRECTORY / f'{filename}.yml'}",
```

Comment:
oh wow, nested f-strings, i didn't know it was possible... what about a simpler one eg
```python
PLAYBOOK_DIRECTORY / f"{filename}.yml"}
```
`PLAYBOOK_DIRECTORY` is a `Path`, so `/` will produce a new path, and the argument will be stringified automatically by `subprocess`, I think